### ISSUE TITLE
Using more test data than training data 

### ISSUE THREAD

**Issue by CunliangGeng** [carpentries-incubator/deep-learning-intro#291]
In Epsode4, we are using 5000 training images, but test set has 10 000 images.

It's better to have less images in test set than training.

**Comment by svenvanderburg** [comment]
Now obselete since #448
